File: versioned_docs/version-1.9.x/examples/js/src/data_replication_remove.mjs
import {Client, QuotaType, ReplicationSettings} from "reduct-js";
import assert from "node:assert";

// Create a client instance
const client = new Client("http://127.0.0.1:8383", {apiToken: "my-token"});

// Remove the `example-replication` replication
await client.deleteReplication("example-replication");

File: versioned_docs/version-1.10.x/examples/py/src/data_replication_remove.py
import asyncio

from reduct import Client, ReplicationSettings


async def main():
    # Create a client instance
    client = Client("http://127.0.0.1:8383", api_token="my-token")
    # Remove the `example-replication` replication
    await client.delete_replication("example-replication")


loop = asyncio.get_event_loop()
loop.run_until_complete(main())
